src/test/ui/pattern/const-pat-ice.stderr

```diff
@@ -10,4 +10,3 @@ note: we would appreciate a bug report: https://github.com/rust-lang/rust/blob/m
 note: rustc VERSION running on TARGET
 
 note: compiler flags: FLAGS
-
```

Comment:
These extra trailing newlines have disappeared. They are necessary though, as the actual output of rustc does contain them. Please add them back.

Reply:
Done. I didn't mean to remove them, my editor did behind my back

Reply:
I assumed that was the case :) My editor did the same thing recently.